custom/icds_reports/tasks.py

```diff
@@ -1855,7 +1857,18 @@ def update_service_delivery_report(target_date):
 
 def update_bihar_api_table(target_date):
     current_month = force_to_date(target_date).replace(day=1)
-    BiharAPIDemographics.aggregate(current_month)
+    _agg_bihar_api_demographics.delay(current_month)
```

Comment:
delay because. I will be kicking it off via airflow only and don't want this to block the aggregation completion time.